guides/v2.1/cloud/project/admin-git-push.md

```diff
@@ -12,7 +12,7 @@ github_link: cloud/project/admin-git-push.md
 # Step by step walk-through of `git push` output
 To better grasp what is happening here, we will show you what the output of the `git push` command might look like with {{site.data.var.ece}}.
 
-For this example, we use a sample PHP project with multiple data backends, MySQL, Redis, and Elasticsearch. It has its PHP dependencies in a `composer.json` file. It uses the Ruby library `sass` to compile the SCSS during its build process.
+For this example, we use a sample PHP project configured with multiple services including MySQL, Redis, and Elasticsearch. It has its PHP dependencies in a `composer.json` file. It uses the Ruby library `sass` to compile the SCSS during its build process.
```

Comment:
I thought you were getting rid of the word "backend" but you're keeping it around as two words. I don't know why you would delete "data" though.

Reply:
@keharper Because MySQL, Redis are data backends -- Elasticsearch isn't -- 
It seemed like the sentence should have been something like with multiple data back ends like MySQL and Redis. 
I thought it was more accurate to say configured with multiple services -- but maybe not?  Thoughts?

Reply:
I made this comment because line 89 mentions five different data back ends, and ES appears to be in that last. Whatever is the most accurate.

Reply:
They are configured as services in the Cloud infrastructure.